java/client/src/main/java/glide/api/models/ClusterValue.java

```diff
@@ -68,6 +69,17 @@ public static <T> ClusterValue<T> ofMultiValue(Map<String, T> data) {
         return res;
     }
 
+    /** A constructor for the value. */
+    public static <T> ClusterValue<T> ofMultiValueBinary(Map<GlideString, T> data) {
```

Comment:
Maybe add a todo - should be removed once we fix creating a cluster response
Consider also updating cluster value tests